fishtank/src/cluster.ts

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Docker } from './backend'
+
+export const DEFAULT_IMAGE = 'ironfish:latest'
+
+export class Cluster {
+  public readonly name: string
```

Comment:
What about `networkName` instead of `name`? I saw this attribute is referred as network name in #4

Reply:
The name of the cluster is used for a multitude of things: to give names to containers, to networks, to volumes, and virtually any resource that you can think of. I choose to use this name as the network name because (at least for now) the plan is to support creating only 1 network, therefore I see no need to add a suffix to it. But if you see this PR, you'll see that the cluster name is used to build the prefix of container names.

Reply:
Good to know you plan for cluster name.